trunk/NDHMS/Routing/Reservoirs/Level_Pool/module_levelpool.F

```diff
@@ -319,7 +331,7 @@ real :: dh, dh1, dh2, dh3      ! Depth in weir, and height function for 3 order
        endif
 
        if(H .ge. maxh) then  ! overtop condition
-          discharge = qi1
+          discharge = qi1 - (((maxh - Htmp) * sap) / dt)
```

Comment:
This is redundant with #292

Reply:
as we noted before...

Reply:
Yes. We needed this fix for our testing too.

Reply:
Right, I'm just making the note.